Cargo.toml

```diff
@@ -23,7 +23,7 @@ cargo-util = { version = "0.2.4", path = "crates/cargo-util" }
 cargo_metadata = "0.14.0"
 clap = "4.2.0"
 core-foundation = { version = "0.9.0", features = ["mac_os_10_7_support"] }
-crates-io = { version = "0.36.1", path = "crates/crates-io" }
+crates-io = { version = "0.37.0", path = "crates/crates-io" }
```

Comment:
Side note: the version bump check in CI didn't work, since the new version in beta hasn't yet published. But in reality, the script should treat any version in beta as “published”.